src/TodoistApi.user.test.ts

```diff
@@ -0,0 +1,103 @@
+import { TodoistApi } from '.'
+import { DEFAULT_AUTH_TOKEN } from './testUtils/testDefaults'
+import { getSyncBaseUri, ENDPOINT_REST_USER } from './consts/endpoints'
+import { setupRestClientMock } from './testUtils/mocks'
+
+function getTarget(baseUrl = 'https://api.todoist.com') {
+    return new TodoistApi(DEFAULT_AUTH_TOKEN, baseUrl)
+}
+
+const DEFAULT_CURRENT_USER_RESPONSE = {
```

Comment:
```suggestion
const DEFAULT_CURRENT_USER_RESPONSE: CurrentUser = {
```

Reply:
@amix Note: this was based on the code I saw before you fixed the linting issue, you'll need to add the type back into the import section of this file.